docs/authentication/social-connections/microsoft.mdx

```diff
@@ -62,7 +62,7 @@ You will be prompted to provide a name and lifetime for the new secret.
 
 ![](/docs/images/authentication-providers/microsoft/08f6445903a233c89ad96be25de3293def891cb6-1172x854.png)
 
-After confirming your choicem the new secret will be listed. This is the only time you will able to copy & paste it, so make sure you do so before navigating away. Also, take note of the lifetime of said secret because you will need to generate a new one before it expires.
+After confirming your choice the new secret will be listed. This is the only time you will able to copy & paste it, so make sure you do so before navigating away. Also, take note of the lifetime of said secret because you will need to generate a new one before it expires.

```

Comment:
```suggestion
After confirming your choice, the new secret will be listed. This is the only time you will able to copy & paste it, so make sure you do so before navigating away. Also, take note of the lifetime of said secret because you will need to generate a new one before it expires.

```